src/components/my-history/custom-my-history.vue

```diff
@@ -101,7 +103,7 @@
                   </div>
 
                   <template #remove-button-content>
-                    <CrossTinyIcon :class="{ 'x-icon--l': $x.device === 'mobile' }" />
+                    <CrossTinyIcon :class="{ 'x-icon-lg': $x.device === 'mobile' }" />
```

Comment:
After talking with @cris-crash, change the component to be lg always:

```suggestion
                    <CrossTinyIcon class="x-icon-lg" />
```